packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -17,7 +17,7 @@ import { SubmenuPosition } from "./MenuItem.Types";
 
 export interface MenuItemProps extends VibeComponentProps {
   title?: string;
-  label?: string;
+  label?: string | React.ReactElement<typeof Label>;
```

Comment:
that's a great solution! I initially thought about adding a labelProps, but that's way better if we already want to allow everything!